python/cudf/cudf/utils/gpu_utils.py

```diff
@@ -1,4 +1,12 @@
 def validate_setup():
+    import os
+
+    # TODO: Remove the following check once we arrive at a solution for #4827
+    # This is a temporary workaround to unblock internal testing
+    # related issue: https://github.com/rapidsai/cudf/issues/4827
+    if "DASK_PARENT" in os.environ:
```

Comment:
To allow us to instruct users to run this manually via Dask lets change this to only have this behavior on import, where we change this and in the `__init__` to `check_dask=False`.
```suggestion
def validate_setup(check_dask=True):
    import os

    # TODO: Remove the following check once we arrive at a solution for #4827
    # This is a temporary workaround to unblock internal testing
    # related issue: https://github.com/rapidsai/cudf/issues/4827
    if not check_dask and "DASK_PARENT" in os.environ:
```